python/cuml/test/test_nearest_neighbors.py

```diff
@@ -232,14 +236,14 @@ def test_knn_separate_index_search(input_type, nrows, n_feats, k, metric):
         D_cuml_arr = D_cuml.as_gpu_matrix().copy_to_host()
         I_cuml_arr = I_cuml.as_gpu_matrix().copy_to_host()
     else:
-        assert isinstance(D_cuml, np.ndarray)
-        assert isinstance(I_cuml, np.ndarray)
-        D_cuml_arr = D_cuml
-        I_cuml_arr = I_cuml
+        assert isinstance(D_cuml, cp.core.core.ndarray)
+        assert isinstance(I_cuml, cp.core.core.ndarray)
+        D_cuml_arr = D_cuml.get()
```

Comment:
D_cuml_arr is now a numpy array, right? That seems like confusing naming?

Reply:
I didn't want to change the name to stay consistent with the cudf path. But in order to avoid any confusion I renamed it to `D_cuml_np`